src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/interfaces/ActivityLoggable.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.sandl.snlevents.actions.interfaces;
+
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.ActivityLog;
+
+import java.util.List;
+
+@SuppressWarnings("squid:S1214")
+public interface ActivityLoggable {
+    String HEARING_ENTITY = "hearing";
```

Comment:
Shouldn't that be a static field in Hearing.java class? Besides, maybe we could use some Java built-in methods to obtain the class name instead of hardcoding the name? That would reduce some typo danger.

Reply:
hymm, the easiest option would be to have enum with Entity names ;)

Reply:
enum or class/entity name enforces tight coupling, where this is may be better to use loose coupling here

Reply:
interface ActivityStatus {
   getEntityName()
}

enum SessionActivityStatus implements ActivityStatus {
   public getEntityName() { return "Session"}
}

If we had ActivityStatuses enums specific to entities we could get the entity name just like that. It also links to my other comment. You wouldn't have to hardcode the entityname in your classes or generic interfaces. Would that make sense?